aptos/example/mint_nft/sources/minting.move

```diff
@@ -0,0 +1,106 @@
+module mint_nft::minting {
+    use std::signer;
+    use std::string::{Self, String};
+    use std::vector;
+
+    use aptos_framework::account;
+    use aptos_framework::event::EventHandle;
+    use aptos_framework::coin;
+    use aptos_framework::aptos_coin;
+
+    use pyth::pyth;
+    use pyth::price_identifier;
+    use pyth::i64;
+    use pyth::price;
+
+    use aptos_std::math64::pow;
+
+    use aptos_token::token::{Self, TokenDataId};
+
+    const APTOS_USD_PRICE_FEED_IDENTIFIER : vector<u8> = x"ca80ba6dc32e08d06f1aa886011eed1d77c77be9eb761cc10d72b7d0a2fd57a6"; // This is actually the ETH/USD while APT is not listed
```

Comment:
Could we call this ETH/USD? And do everything in ETH? It's confusing to have the wrong price ID here.

Reply:
Added a comment

Reply:
Let's update when APT price feeds comes out